examples/ConfigurableFirmata/ConfigurableFirmata.ino

```diff
@@ -178,6 +178,9 @@ OneWireFirmata oneWire;
 #include <StepperFirmata.h>
 StepperFirmata stepper;
 
+#include <StepperFirmata2.h>
+StepperFirmata2 stepper2;
```

Comment:
After renaming class to AccelStepperFirmata, change the instance name to accelStepper.